app/views/file_upload.scala.html

```diff
@@ -61,7 +61,7 @@ <h1 id="header" class="heading-xlarge">@messages("file.upload.page.header")</h1>
 
                 <input id="choose-file" name="file" type="file" size="40" data-journey-click="button - click:Upload a file:Choose file"/>
                 <p class="ras-bottom-margin"><a href="http://www.gov.uk/guidance/find-the-relief-at-source-residency-statuses-of-multiple-members" target="_blank" id="upload-help-link" data-journey-click="link - click:Upload a file:Get help formatting your file">@messages("get.help.uploading.link")</a></p>
-                <button class="button" type="submit" id="continue" data-journey-click="button - click:Upload a file:Continue">@messages("continue")</button>
+                <button class="button" type="submit" id="continue" data-journey-click="button - click:Upload a file:Continue" data-ignore-double-submit="true">@messages("continue")</button>
```

Comment:
not sure this is the best way to fix this issue, we should still try to prevent double clicks